android-components/components/lib/dataprotect/src/main/java/mozilla/components/lib/dataprotect/Keystore.kt

```diff
@@ -157,8 +158,11 @@ open class Keystore(
         }
     }
 
-    private fun getKey(): SecretKey? =
+    private fun getKey(): SecretKey? = try {
         wrapper.getKeyFor(label) as? SecretKey?
+    } catch (e: UnrecoverableKeyException) {
+        null
```

Comment:
Let's add a tagged Logger for this exception.

```
private val logger = Logger("KeyStore")
logger.error("Something", e)
```

Reply:
Agreed.  I'll add the log, thanks